gitlab/README.md

```diff
@@ -10,6 +10,12 @@ See [Monitoring GitLab with Prometheus][1] for more information.
 
 ## Setup
 
+### Prerequisites
+
+This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see the [Latest and Legacy Versioning For OpenMetrics-based Integrations][17].
```

Comment:
I'm not sure about giving this such a prominent place (its own section and at the beginning of the setup). I also wouldn't think of this as a "prerequisite". What would you think of turning this into some sort of note (perhaps one of those colored `div`s?), probably placed somewhere in the `##### Metric collection` subsection below? Note that there's also a reference to a "legacy example" on that first point as well.

https://github.com/DataDog/integrations-core/blob/86a09fc3836b9145bf97d22ea2fc19c582cff587/gitlab/README.md?plain=1#L32-L35

This is because the most obvious way in which the reader is expected to make use of this information is either when setting it up or when looking for specific metrics that they might be expecting.

Reply:
We introduce the Prerequisites section across all the integration READMEs 🤔